src/cluster.c

```diff
@@ -1918,7 +1918,8 @@ int clusterProcessPacket(clusterLink *link) {
         type == CLUSTERMSG_TYPE_MEET)
     {
         serverLog(LL_DEBUG,"%s packet received: %p",
-            type == CLUSTERMSG_TYPE_PING ? "ping" : "pong",
+            type == CLUSTERMSG_TYPE_PING ? "ping" :
+                type == CLUSTERMSG_TYPE_PONG ? "pong" : "meet",
```

Comment:
```suggestion
serverLog(LL_DEBUG,"%s packet received: %p",
    clusterGetMessageTypeString(type),
    (void*)link->node);
```

Reply:
Also, should (void*)link->node be replaced with the node ID? I never understood why it returned the raw pointer.

Reply:
@madolson I think node ID is more meaningfull.
And I think should we delete this line?
https://github.com/redis/redis/blob/f61c37cec900ba391541f20f7655aad44a26bafc/src/cluster.c#L1862
They are redundant.

Reply:
Yeah, we should be able to delete that line.